doc/releases.md

```diff
@@ -0,0 +1,288 @@
+  * [⚠️ Merge the backporting PR](#merge-the-backporting-pr)
+  * [⚠️ Publish a pre-release](#publish-a-pre-release)
+  * [Announce the release candidate](#send-an-announcement-email)
+  * [Check for security updates](#check-for-security-updates)
+* [LTS Release](#lts-release)
+  * [Announce the start](#announce-the-start-of-the-lts-process)
+  * [⚠️ Start the build](#start-the-lts-build)
+  * [⚠️ Publish the GitHub release](#publish-the-github-release)
+  * [Confirm acceptance tests](#confirm-acceptance-tests)
+  * [Update Jira issues](#update-jira-issues)
+  * [⚠️ Publish Docker image](#publish-docker-image)
+  * [Check docker images](#check-docker-images)
+  * [Prepare Jenkins infrastructure for updates](#prepare-infrastructure-for-the-next-release)
+  * [Visit changelog](#visit-changelog)
+
+## Glossary
+This document uses the following generic terms, which may need a clarification:
+
+- checklist: The [release checklist](https://github.com/jenkins-infra/release/blob/master/.github/ISSUE_TEMPLATE/1-lts-release-checklist.md)
+- 2.VVV: The version number of the weekly release the LTS release is based upon, e.g. 2.375.  
+  2.VVV.x translates to 2.375.x in this example, .x is no placeholder.
+- RC: Release Candidate
+- ⚠️: Specific permission required. See [Access Prerequisites](#access-prerequisites).
+
+## Software Prerequisites:
+- git 2.38.1 or newer
+- bash 5.2 or newer
+  - groovy 2.x or newer available in your local packages
+  - xdg-utils in your local packages (to be removed in the future, not available on homebrew/macOS)
+- maven 3.6.3 or newer
+- openjdk 11 or 17
+
+## Knowledge Prerequisites:
+- Novice knowledge about GitHub, how to create branches, pull requests and releases.
+- Novice knowlege how to use a terminal to run a script and specify an input.
+- Competence knowlege how to cherry-pick commits and resolve merge conflicts with a situational merge strategy.
+- Beginner knowledge about Jenkins, multibranch pipelines and folders.
+
+## Access Prerequisites:
+**The following is only available to release team members.**  
+You can safely ignore it, these are not needed for the majority of steps. If they are needed, the steps below will tell you with a warning symbol (⚠️).  
+It's recommended to ask someone from the release team to take care of these steps, if you are not a release team member yourself.
+<details>
+<summary>Access to the Jenkins release infrastructure</summary>
+
+- An account on [accounts.jenkins.io](https://accounts.jenkins.io/).
+    - The account is part of the `release-core` LDAP group.
+- A VPN client to access the Jenkins VPN to access non-public resources.
+- Access to [release.ci.jenkins.io](https://release.ci.jenkins.io/) and [trusted.ci.jenkins.io](https://trusted.ci.jenkins.io:1443).
+- Write access on [jenkinsci/jenkins](https://github.com/jenkinsci/jenkins), [jenkinsci/packaging](https://github.com/jenkinsci/packaging) and [jenkins-infra/release](https://github.com/jenkins-infra/releases).
```

Comment:
possibly refer to release team instead (https://github.com/orgs/jenkinsci/teams/release) although it will 404 for people who aren't members of the GitHub org